pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -1008,7 +1008,7 @@ https://github.com/GnikDroy/projections.nvim/,HEAD,
 https://github.com/kevinhwang91/promise-async/,HEAD,
 https://github.com/frigoeu/psc-ide-vim/,,
 https://github.com/Shougo/pum.vim/,HEAD,
-https://github.com/shaunsingh/moonlight.nvim/,,pure-lua
+https://github.com/shaunsingh/moonlight.nvim/,,
```

Comment:
The updater script would move this to another place, just run `nix-build -A vimPluginsUpdater; ght ./result/bin/vim-plugins-updater update 'vimPlugins.moonlight-nvim' --no-commit` and commit whatever changes it does